integration/bench/load_test.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	k6Image = "loadimpact/k6:0.37.0"
+	k6Image = "loadimpact/k6:latest"
```

Comment:
> Just a question for learning purposes. If we have the k6 version pinned here, why our bench tests started failing due changes in a newer version?

It was pinned to un-break them until we had a fix 😛

> This is related to some doubts about changing to latest here as it can cause problems in the future if a future version also includes changes that break our tests.

True. But if we pin it we will probably never update it. It's very well hidden within the code to get updated. IMO it's worth the occasional fix not to drift from the latest release.

Reply:
agreed. Thanks for the context!